testsuite/forge-cli/src/main.rs

```diff
@@ -683,6 +691,12 @@ fn land_blocking_test_suite(duration: Duration) -> ForgeConfig<'static> {
             } else {
                 60
             })),
+            Some(SystemMetricsThreshold::new(
+                // Check that we don't use more than 12 CPU cores for 30% of the time.
+                MetricsThreshold::new(12, 30),
+                // Check that we don't use more than 5 GB of memory for 30% of the time.
+                MetricsThreshold::new(5 * 1024 * 1024 * 1024, 30),
```

Comment:
would this still fail our recent failures like: https://github.com/aptos-labs/aptos-core/actions/runs/3032273307

```
Test Failed: Metrics violated threshold, max_breach_pct 30, breach_pct43 
```